tests/Doctrine/Tests/Models/EagerBatched/Article.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: wouter
+ * Date: 11/18/15
+ * Time: 1:00 PM
+ */
+
+namespace Doctrine\Tests\Models\EagerBatched;
+use Doctrine\ORM\Mapping\Table;
+
+/**
+ * Class Article
+ * @package Doctrine\Tests\Models\EagerBatched
+ *
+ * @Entity()
+ * @Table(name="eager_batched_article")
+ */
+class Article
+{
+
+	/**
+	 * @Id @Column(type="integer", name="id") @GeneratedValue
+	 */
+	protected $id;
+
+	/**
+	 * @Column(type="string")
+	 */
+	protected $title;
+
+	/**
+	 * @OneToMany(targetEntity="Tag",mappedBy="article",cascade={"persist"},fetch="EAGER_BATCHED")
+	 */
+	protected $tags;
+
+	/**
+	 * Get id
+	 *
+	 * @return mixed
+	 */
+	public function getId() {
+		return $this->id;
+	}
+
+	/**
+	 * Set id
+	 *
+	 * @param mixed $id
+	 * @return $this
+	 */
+	public function setId($id) {
+		$this->id = $id;
+
+		return $this;
+	}
+
+	/**
+	 * Get title
+	 *
+	 * @return mixed
```

Comment:
title is `type="string`, so I'd expect either `@return string` or `@return string|null` if field is nullable. And all methods must conform to PSR-2 (opening bracket on next line)